src/ethereum_spec_tools/new_fork.py

```diff
@@ -58,7 +59,8 @@ def find_replace(dir: str, find: str, replace: str, file_pattern: str) -> None:
             file_path = os.path.join(path, filename)
             with open(file_path, "r+b") as f:
                 s = f.read()
-                s = s.replace(find.encode(), replace.encode())
+                find_pattern = (r"\b" + find + r"\b").encode()
```

Comment:
Probably unnecessary for the most part, but:

```suggestion
                find_pattern = (r"\b" + re.escape(find) + r"\b").encode()
```